Makefile

```diff
@@ -43,3 +51,7 @@ test-e2e-gitlab: tidy fmt vet
 test-e2e-stash: tidy fmt vet
 	go test ${TEST_FLAGS} -race -coverprofile=coverage.txt -covermode=atomic -tags=e2e ./stash/...
 
+test-e2e-gitea: tidy fmt vet
```

Comment:
I tried this locally and all tests are failing. Gitea container logs show this when the tests are running:

```
2023/06/05 09:08:30 [647da60e-24] router: failed    GET /api/v1/orgs/fluxcd-testing/teams?limit=0&page=1 for 172.19.0.1:32810, panic in 0.6ms @ v1/api.go:446(v1.reqOrgMembership), err=runtime error: invalid memory address or nil pointer dereference
2023/06/05 09:08:30 ...common/middleware.go:72:1() [E] [647da60e-24] PANIC: runtime error: invalid memory address or nil pointer dereference
        /usr/local/go/src/runtime/panic.go:260 (0x458adc)
[...]
```

Reply:
if you used the docker compose to spin up your instance, the token is located at `/tmp/gitea-token`. You should use it to run the test.

Reply:
Explicitly setting this shouldn't be necessary as it's the default, anyway. That's how the GitLab tests work.